src/frontend/get-github-subscriptions.ts

```diff
@@ -2,11 +2,11 @@ import { Subscription } from "../models";
 import { NextFunction, Request, Response } from "express";
 
 export default async (req: Request, res: Response, next: NextFunction): Promise<void> => {
-	if (!req.session.githubToken) {
+	const { github, client, isAdmin, githubToken } = res.locals;
```

Comment:
I think this should be feature flagged so we could keep reading from session if something is wrong... Am I missing something?

Reply:
Discussed in slack, we are not feature flagging this change